ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fhir/FHIRMaskingProvider.java

```diff
@@ -1,70 +1,90 @@
 /*
- * (C) Copyright IBM Corp. 2016,2021
+ * (C) Copyright IBM Corp. 2016,2022
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 package com.ibm.whc.deid.providers.masking.fhir;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 import java.util.stream.Collectors;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.ibm.whc.deid.ObjectMapperFactory;
-import com.ibm.whc.deid.providers.masking.AbstractComplexMaskingProvider;
 import com.ibm.whc.deid.providers.masking.ComplexMaskingProvider;
 import com.ibm.whc.deid.providers.masking.MaskingProviderFactory;
+import com.ibm.whc.deid.pruners.GlobalProcessorFactory;
 import com.ibm.whc.deid.shared.exception.KeyedIllegalArgumentException;
 import com.ibm.whc.deid.shared.exception.KeyedRuntimeException;
 import com.ibm.whc.deid.shared.pojo.config.DeidMaskingConfig;
+import com.ibm.whc.deid.shared.pojo.config.GlobalProcessorConfig;
 import com.ibm.whc.deid.shared.pojo.config.json.JsonConfig;
 import com.ibm.whc.deid.shared.pojo.config.json.JsonMaskingRule;
 import com.ibm.whc.deid.shared.pojo.masking.ReferableData;
+import com.ibm.whc.deid.shared.pojo.masking.ReferableNode;
 import com.ibm.whc.deid.utils.log.LogCodes;
 import com.ibm.whc.deid.utils.log.Messages;
-import scala.Tuple2;
 
-public class FHIRMaskingProvider extends AbstractComplexMaskingProvider
-    implements ComplexMaskingProvider {
+public class FHIRMaskingProvider implements ComplexMaskingProvider, Serializable {
 
-  private static final long serialVersionUID = 5945527984023679481L;
+  private static final long serialVersionUID = 1L;

```

Comment:
Wouldn't it be safer to have a more random number here ?

Reply:
For what we are doing, the value shouldn't matter.  I changed the value since I made changes to the class members, but I wouldn't have need to do that since we aren't storing serialized versions long term.

Reply:
ok. Thx